org.eclipse.xtext.xtext.ui.examples/pom.xml

```diff
@@ -12,12 +12,31 @@
 
 	<build>
 		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-clean-plugin</artifactId>
+				<version>3.0.0</version>
+				<configuration>
+					<phase>initialize</phase>
```

Comment:
why not on the "clean" phase?

Reply:
sry this was just an exp. did cont want to change the phase at all